src/main/java/com/klarna/hiverunner/HiveShell.java

```diff
@@ -196,5 +197,13 @@
      */
     OutputStream getResourceOutputStream(String targetFile);
 
+    /**
+     * Returns an {@link InsertIntoTable} that allows programmatically inserting data into a table in a fluent manner.
+     * <p/>
+     * May only be called post #start()
+     * @param databaseName The database name
+     * @param tableName The table name
+     */
+    InsertIntoTable insertInto(String databaseName, String tableName);
```

Comment:
Could this factory method be moved to it's own package instead?

Since the only actual point of contact is the hiveconf there is no real need for a factory
method in the hive shell.

If the factory method could take the HiveShell then we have the option to move the lifecycle 
of for instance the HCatClient without affecting the API of InsertIntoTable

E.g: 
```
public class InsertIntoTable {
...
	public static InsertIntoTable newInstance(String databaseName, String tableName, HiveShell hiveShell) {
		...
        return new InsertIntoTable(builder, inserter);
    }
...
}
```